examples/game_of_life/.serviceignore

```diff
@@ -1,4 +1,4 @@
-# This file specifies that when we publish this plugin to a Plug-in Library,
+# This file specifies that when we publish this plugin to a Plug-In Library,
```

Comment:
Shouldn't we use "plug-in" for the first usage too?

I think we don't have to capitalize "plug-in library" here. https://www.ni.com/docs/en-US/bundle/measurementplugins/page/plugin-libs.html uses lowercase when referring to it in English text.
```suggestion
# This file specifies that when we publish this plug-in to a plug-in library,
```